conf/.env

```diff
@@ -0,0 +1,379 @@
+#LDAP_BACKGROUND_SYNC_INTERVAL=100
+LDAP_BACKGROUND_SYNC_INTERVAL=100
+
+# LDAP_BACKGROUND_SYNC_KEEP_EXISTANT_USERS_UPDATED :
+# example :  LDAP_BACKGROUND_SYNC_KEEP_EXISTANT_USERS_UPDATED=true
+#LDAP_BACKGROUND_SYNC_KEEP_EXISTANT_USERS_UPDATED=false
+LDAP_BACKGROUND_SYNC_KEEP_EXISTANT_USERS_UPDATED=true
+
+# LDAP_BACKGROUND_SYNC_IMPORT_NEW_USERS :
+# example :  LDAP_BACKGROUND_SYNC_IMPORT_NEW_USERS=true
+#LDAP_BACKGROUND_SYNC_IMPORT_NEW_USERS=false
+LDAP_BACKGROUND_SYNC_IMPORT_NEW_USERS=true
+
+# LDAP_ENCRYPTION : If using LDAPS
+# example :  LDAP_ENCRYPTION=ssl
+#LDAP_ENCRYPTION=false
+
+# LDAP_CA_CERT : The certification for the LDAPS server. Certificate needs to be included in this docker-compose.yml file.
+# example :  LDAP_CA_CERT=-----BEGIN CERTIFICATE-----MIIE+zCCA+OgAwIBAgIkAhwR/6TVLmdRY6hHxvUFWc0+Enmu/Hu6cj+G2FIdAgIC...-----END CERTIFICATE-----
+#LDAP_CA_CERT=
+
+# LDAP_REJECT_UNAUTHORIZED : Reject Unauthorized Certificate
+# example :  LDAP_REJECT_UNAUTHORIZED=true
+#LDAP_REJECT_UNAUTHORIZED=false
+
+# LDAP_USER_SEARCH_FILTER : Optional extra LDAP filters. Don't forget the outmost enclosing parentheses if needed
+# example :  LDAP_USER_SEARCH_FILTER=
+#LDAP_USER_SEARCH_FILTER=
+LDAP_USER_SEARCH_FILTER="(objectclass=posixAccount)"
+
+# LDAP_USER_SEARCH_SCOPE : base (search only in the provided DN), one (search only in the provided DN and one level deep), or sub (search the whole subtree)
+# example :  LDAP_USER_SEARCH_SCOPE=one
+#LDAP_USER_SEARCH_SCOPE=
+LDAP_USER_SEARCH_SCOPE=sub
+
+# LDAP_USER_SEARCH_FIELD : Which field is used to find the user
+# example :  LDAP_USER_SEARCH_FIELD=uid
+#LDAP_USER_SEARCH_FIELD=
+LDAP_USER_SEARCH_FIELD=uid
+
+# LDAP_SEARCH_PAGE_SIZE : Used for pagination (0=unlimited)
+# example :  LDAP_SEARCH_PAGE_SIZE=12345
+#LDAP_SEARCH_PAGE_SIZE=0
+LDAP_SEARCH_PAGE_SIZE=0
+
+# LDAP_SEARCH_SIZE_LIMIT : The limit number of entries (0=unlimited)
+# example :  LDAP_SEARCH_SIZE_LIMIT=12345
+#LDAP_SEARCH_SIZE_LIMIT=0
+LDAP_SEARCH_SIZE_LIMIT=0
+
+# LDAP_GROUP_FILTER_ENABLE : Enable group filtering
+# example :  LDAP_GROUP_FILTER_ENABLE=true
+#LDAP_GROUP_FILTER_ENABLE=false
+LDAP_GROUP_FILTER_ENABLE=true
+
+# LDAP_GROUP_FILTER_OBJECTCLASS : The object class for filtering
+# example :  LDAP_GROUP_FILTER_OBJECTCLASS=group
+#LDAP_GROUP_FILTER_OBJECTCLASS=
+LDAP_GROUP_FILTER_OBJECTCLASS=posixGroup
+
+# LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE :
+# example :
+#LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE=
+LDAP_GROUP_FILTER_GROUP_ID_ATTRIBUTE=cn
+
+# LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE :
+# example :
+#LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE=
+LDAP_GROUP_FILTER_GROUP_MEMBER_ATTRIBUTE=memberUid
+
+# LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT :
+# example :
+#LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT=
+LDAP_GROUP_FILTER_GROUP_MEMBER_FORMAT=uid
+
+# LDAP_GROUP_FILTER_GROUP_NAME :
+# example : LDAP_GROUP_FILTER_GROUP_NAME=wekan_user
+#LDAP_GROUP_FILTER_GROUP_NAME=
+LDAP_GROUP_FILTER_GROUP_NAME=sftpusers
```

Comment:
Uh is it really the group we wanna use ? o.O

Reply:
Well, it has to be a group available in open ldap... and by default there are only two... sftpusers with all yunohost users and admins...

When group permissions based on ldap groups and a ldap helper will be available , we will be able to create a dedicated group $app during install and replace sftpusers